### PR TITLE
update playbooks_variables.rst

### PR DESCRIPTION
**Pull request by Dule-martins**
Added a section with a table displaying Booleans and Values.

##### SUMMARY

This contains a table of boolean and values accepted by ansible.

##### ISSUE TYPE

- Docs Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: docs/docsite/rst/playbook_guide/playbooks_variables.rst [hunk 1]
```diff
@@ -97,6 +97,19 @@ You will see: ``ERROR! Syntax Error while loading YAML.`` If you add quotes, Ans
 
 .. _list_variables:
 
+Boolean variables
+==============
```
Comment (webknjaz) [inline docs/docsite/rst/playbook_guide/playbooks_variables.rst]:
```suggestion
=================
```

File: docs/docsite/rst/playbook_guide/playbooks_variables.rst [hunk 2]
```diff
@@ -97,6 +97,19 @@ You will see: ``ERROR! Syntax Error while loading YAML.`` If you add quotes, Ans
...
+
+Ansible accepts a broad range of values for boolean variables: ``true/false``, ``1/0``, ``yes/no``, ``True/False``, and so on.
+While documentation examples focus on ``true/false`` to be compatible with ``ansible-lint`` default settings, you can use any of the following:
```
Comment (webknjaz) [inline docs/docsite/rst/playbook_guide/playbooks_variables.rst:104]: Do the all the examples really use true/false? :thinking:

File: docs/docsite/rst/playbook_guide/playbooks_variables.rst [hunk 3]
```diff
@@ -97,6 +97,19 @@ You will see: ``ERROR! Syntax Error while loading YAML.`` If you add quotes, Ans
...
+
+ ====================== ====================================================================
+    Boolean                         Value
+ ====================== ====================================================================
+    BOOLEANS_TRUE          ('y', 'yes', 'on', '1', 'true', 't', 1, 1.0, True)
```
Comment (webknjaz) [inline docs/docsite/rst/playbook_guide/playbooks_variables.rst:109]: Are the parentheses necessary? And occasional quotations?

Comment (webknjaz) [reply docs/docsite/rst/playbook_guide/playbooks_variables.rst:109]: Looking at the other tables, I'd say that a lot of double backticks are missing here.

File: docs/docsite/rst/playbook_guide/playbooks_variables.rst (hunk 2 above)
Comment (Dule-martins) [reply docs/docsite/rst/playbook_guide/playbooks_variables.rst:104]: @samccann could have more information about that

File: docs/docsite/rst/playbook_guide/playbooks_variables.rst (hunk 3 above)
Comment (Dule-martins) [reply docs/docsite/rst/playbook_guide/playbooks_variables.rst:109]: You are suggesting I add the double backticks?

Comment (webknjaz) [reply docs/docsite/rst/playbook_guide/playbooks_variables.rst:109]: Yes, follow the example above in the document.

File: docs/docsite/rst/playbook_guide/playbooks_variables.rst (hunk 2 above)
Comment (samccann) [reply docs/docsite/rst/playbook_guide/playbooks_variables.rst:104]: It's a bit of a gray area. On the RST files we control, we hope to get most examples using true/false. For examples within collections, we can only suggest they follow that standard but we can't force it.

**Comment by samccann** [comment]
Looks like we had two contributors adding this section. Since the other PR was ready to merge, I merged it. Thanks @Dule-mart for your work on this PR. Closing this in favor of the one that was merged.

**Pull request closed without merging**